src/shipit_code_coverage/shipit_code_coverage/codecov.py

```diff
@@ -147,22 +147,25 @@ def rewrite_jsvm_lcov(self, zip_file_path):
             while not self.build_finished:
                 self.build_finished_cv.wait()
 
-        out_dir = zip_file_path[:-4]
+        out_name = zip_file_path[:-4]
+        out_dir = out_name
+        out_file = out_name + '.info'
 
         zip_file = zipfile.ZipFile(zip_file_path, 'r')
         zip_file.extractall(out_dir)
         zip_file.close()
 
-        lcov_files = [os.path.abspath(os.path.join(out_dir, f)) for f in os.listdir(out_dir)]
-        run_check(['gecko-env', './mach', 'python', 'python/mozbuild/mozbuild/codecoverage/lcov_rewriter.py'] + lcov_files, cwd=self.repo_dir)
+        run_check([
+            'gecko-env', './mach', 'python',
+            'python/mozbuild/mozbuild/codecoverage/lcov_rewriter.py',
+            os.path.abspath(out_dir),
+            '--output-file', os.path.abspath(out_file)
```

Comment:
Is this `.info` output file actually used anywhere, or will it be?

Reply:
It is used in `generate_info` below, which uses `os.listdir` to find all the info files (rewritten, coming from JS) and the ZIP files (not rewritten, coming from C++).